custom_components/tuya_local/devices/thermoval-aquati-waterheater.yaml

```diff
@@ -0,0 +1,92 @@
+name: Electric Water Heater
+products:
+  - id: jxma5t0bc0sagddh
+    manufacturer: Thermoval
+    model_id: "WS-42S-2"
+    model: "Aqua Ti Water Heater"
+entities:
+  - entity: water_heater
+    translation_key: water_air
+    dps:
+      - id: 1
+        type: boolean
+        name: water_heater_mode
+        mapping:
+          - dps_val: false
+            value: off
```

Comment:
The "off" needs to be quoted to avoid yaml interpreting it as a boolean and the parser converting it to Python `False`.